### DISCUSSION TITLE
Feature Request: VSCE: quick fix for type error

### DISCUSSION THREAD

**Discussion by PabloLION** [microsoft/pylance-release#2675] [Enhancement]
- [x] searched "quick fix" and checked open issues.

## Request
Can we add this "ignore type checking in this line" to the "quick fix" function of vscode extension, like eslint does?

## Reason
In https://github.com/microsoft/pylance-release/discussions/1629 it's mentioned that the PEP484 gives an `# type: ignore` option to ignore the type check.

## Thanks

**Comment by judej** [comment]
Thanks for the request. This is not something that is currently supported, but will reevaluate based on upvotes or community feedback.

**Comment by PabloLION** [comment]
Hi @judej. Thanks for the reply. I'm not sure what is "currently supported"? The `ESLint` extension has this in the refactor function, to add a `// eslint-disable-next-line` to the code. I think it's supported by VSCode.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
